library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -132,6 +146,8 @@
   protected transient long currentWindowId;
   protected final transient LinkedList<RecoveryEntry> currentWindowRecoveryState = Lists.newLinkedList();
   protected int operatorId; //needed in partitioning
+  protected int fileCt;
```

Comment:
Suggest fileCount instead of fileCt (abbreviated or cryptic name are OK if used as local variables within a small context of a few lines; for variables with a larger scope or lifetime, longer more descriptive names are preferable).